src/test/java/com/solvd/twitter/unit/FollowerServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package com.solvd.twitter.unit;
```

Comment:
@vpetrovaa your test package names should be the same as package names for production classes

example:
`src/main/java/org/example/foo/Foo.java`
`src/test/java/org/example/foo/FooTest.java`

Reply:
@h1alexbel, ok, fixed https://github.com/vpetrovaa/twitter/pull/16/commits/da3063fb25f3372d02ce8bff17aee04755a5ebc4